src/index.ts

```diff
@@ -45,7 +45,7 @@ export = class OutputDcatUs11 {
         dcatConfig = _.get(siteModel, 'data.feeds.dcatUS11');
       }
 
-      const { stream: dcatStream, dependencies } = getDataStreamDcatUs11(hostname, dcatConfig);
```

Comment:
I believe this still could break the changes I made to use the hostname when the custom domain and the site item's url do not match, just as a heads up.

Reply:
@drspacemanphd thanks for catching that! I made changes accordingly.

Reply:
Thanks!